### PR TITLE
[HttpFoundation] Workaround HHVM rewriting HTTP response line

### PR DESCRIPTION
**Pull request by nicolas-grekas**
| Q | A |
| --- | --- |
| Bug fix? | yes |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | #16481, #9365 |
| License | MIT |
| Doc PR | - |

### CODE REVIEW

**Comment by xabbuh** [comment]
LGTM :+1:

**Comment by nicolas-grekas** [comment]
This can be done on 2.3, closing in favor of #16676

File: src/Symfony/Component/HttpFoundation/Response.php [hunk 1]
```diff
@@ -329,16 +329,16 @@ public function sendHeaders()
...
         }
 
-        // status
-        header(sprintf('HTTP/%s %s %s', $this->version, $this->statusCode, $this->statusText), true, $this->statusCode);
-
         // headers
         foreach ($this->headers->allPreserveCase() as $name => $values) {
             foreach ($values as $value) {
-                header($name.': '.$value, false, $this->statusCode);
+                header($name.': '.$value, false);
```
Comment (xabbuh) [inline src/Symfony/Component/HttpFoundation/Response.php]: Shouldn't we still do this in 2.7+?

Comment (nicolas-grekas) [inline src/Symfony/Component/HttpFoundation/Response.php]: no need, the later header will overwrite any value set here

Comment (nicolas-grekas) [inline src/Symfony/Component/HttpFoundation/Response.php]: oh, yes now I understand. Should be done while merging the 2.3 PR

Comment (nicolas-grekas) [inline src/Symfony/Component/HttpFoundation/Response.php]: See 0a40b54

**Pull request closed without merging**